lib/maths/CStatisticalTests.cc

```diff
@@ -91,6 +94,9 @@ double CStatisticalTests::rightTailFTest(double x, double d1, double d2) {
     if (std::isinf(x)) {
         return 0.0;
     }
+    if (std::isnan(x)) {
+        return 0.0;
```

Comment:
I think it would be safer to make this `1.0`. This is likely due to a 0/0 problem in the calling code. Returning `1.0` here means we shouldn't accept any decision using the F statistic.

Reply:
Good advice, thanks Tom